compiler/tests/evm_contracts.rs

```diff
@@ -188,6 +188,20 @@ fn return_u256() {
     })
 }
 
+#[test]
+fn return_product_u256() {
+    with_executor(&|mut executor| {
+        let harness = deploy_contract(&mut executor, "return_product_u256.vy", "Foo");
+
+        harness.test_function(
+            &mut executor,
+            "bar",
+            vec![u256_token(42), u256_token(42)],
+            Some(u256_token(84)),
+        )
+    })
+}
```

Comment:
@g-r-a-n-t regarding #37 Instead of repeating the above test for multiplication, substraction etc, we could use parameters like this to put it all in one test.

```
#[rstest(input, expected,
    case("return_addition_u256.vy", vec![42, 42, 84]),
    ...
    case("return_substraction_u256.vy", vec![42, 42, 0),
    ...
    case("return_multiplication_u256.vy", vec![5, 5, 25),
    ...
)]
```

Reply:
At least, that is what I would do if this was Py-EVM / Trinity code. Interested to hear your thoughts..

Reply:
I think it looks good.

I also wonder if we could do the same thing here #29, where we're trying to automatically generate tests with macros (which doesn't work anymore) :thinking: